packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added new prop to apply column highlighting style intended for timelines
```

Comment:
```suggestion
  * Added new prop `columnHighlightId` to highlight the column with style intended for timelines application
```

Reply:
[Changed](https://github.com/cerner/terra-clinical/pull/796/commits/d61cc802366b240b410fd07f004d522d32624e81#diff-620a913a25bb7b936559a229e6265d4690812a154b1c23d3479918015309c39dR6) to Neil's suggestion, as it makes more clear